orchagent/port/porthlpr.cpp

```diff
@@ -118,6 +118,14 @@ static const std::unordered_map<std::string, Port::Role> portRoleMap =
     { PORT_ROLE_DPC, Port::Role::Dpc }
 };
 
+static const std::unordered_map<std::string, sai_port_loopback_mode_t> portLoopbackModeMap = {
+    {"none",      SAI_PORT_LOOPBACK_MODE_NONE},
+    {"phy_local", SAI_PORT_LOOPBACK_MODE_PHY},
```

Comment:
@royyi8 How is it different from `SAI_PORT_INTERNAL_LOOPBACK_MODE_PHY`

Reply:
Same reason as above. `SAI_PORT_LOOPBACK_MODE_PHY` is the corresponding attribute for `SAI_PORT_INTERNAL_LOOPBACK_MODE_PHY` in the `sai_port_loopback_mode_t` enum.